app/lib/hosting_environment.rb

```diff
@@ -1,11 +1,37 @@
 # frozen_string_literal: true
 
 module HostingEnvironment
+  ENVIRONMENT_COLOUR = {
+    production: "blue",
+    development: "white",
+    review: "purple",
+    test: "red",
+    qa: "orange",
+    preview: "yellow"
+  }.freeze
+
+  ENVIRONMENT_THEME_COLOUR = {
+    production: "#005eb8",
+    development: "#fff",
+    review: "#d6cce3",
+    test: "#f7d4d1",
+    qa: "#ffdc8e",
+    preview: "#fff59d"
+  }.freeze
+
   class << self
     def name
       pull_request ? "review" : ENV.fetch("SENTRY_ENVIRONMENT", "development")
     end
 
+    def colour
+      ENVIRONMENT_COLOUR[name.to_sym]
```

Comment:
```suggestion
      ENVIRONMENT_COLOUR.fetch(name.to_sym, "white")
```

I suspect you'll need this for the tests to pass.